JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeRoute.kt

```diff
@@ -112,18 +106,12 @@ fun HomeRoute(
         }
     }
 
-    val homeScreenType = getHomeScreenType(windowSize, uiState)
-
-    val showTopAppBar = when (windowSize) {
-        WindowSize.Compact -> true
-        WindowSize.Medium, WindowSize.Expanded -> false
-    }
-
+    val homeScreenType = getHomeScreenType(isDrawerActive, uiState)
     when (homeScreenType) {
         HomeScreenType.FeedWithArticleDetails -> {
             HomeFeedWithArticleDetailsScreen(
                 uiState = uiState,
-                showTopAppBar = showTopAppBar,
+                showTopAppBar = isDrawerActive,
```

Comment:
I'm a bit confused that you're passing on `isDrawerActive` instead of something like `isLargeScreen`. What's the logic you want to bring across? That the home screen type and top app bar visibility depend on whether the drawer is active, or on whether the screen is large? I'd say that second explanation feels more reasonable. So I'd consider passing a boolean called `isExpandedScreen`?

Reply:
`isExpandedScreen` might be a better name now that we don't pass `windowSize`. WDYT @alexvanyo ? I'm not a fan of `showTopAppBar` because it doesn't actually represent what's going on

Reply:
Changed the PR to use `isExpandedScreen` for the top level components + routes. Then move to a more descriptive name when made sense. Thanks!